synapse/lib/cell.py

```diff
@@ -2168,6 +2168,9 @@ async def execmain(cls, argv, outp=None):
         if outp is None:
             outp = s_output.stdout
 
+        if not int(os.getenv('CELL_CHECKHOST', '1')):
```

Comment:
What should this env var really be named?

Reply:
1. we have a envbool helper in synapse.common
2. SYN_CHECKHOST ?

Reply:
This is better done inside of ``initFromArgv`` to allow for picking up on structured logging